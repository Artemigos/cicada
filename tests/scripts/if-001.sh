echo hi

if echo foo | grep -iq o
    echo found foo
fi

if echo foo | grep -iq ar
    echo found bar
fi

if echo foo | grep -iq ar
    echo found foo
else if echo baz | grep -iq az
    echo found baz
else
    echo no foo and no baz
fi

if echo foo | grep -iq a
    echo found a
else if echo foo | grep -iq b
    echo found b
else
    echo no a and no b
fi

echo bye
