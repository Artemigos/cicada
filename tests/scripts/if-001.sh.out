hi
found foo
found baz
no a and no b
bye
